Makefile

```diff
@@ -75,3 +76,21 @@ push-tag:
 	@[ "${TAG}" ] || ( echo ">> env var TAG is not set"; exit 1 )
 	@echo "Pushing tag ${TAG}"
 	@git push git@github.com:open-telemetry/opentelemetry-collector-releases.git ${TAG}
+
+.PHONY: go-licenses
+go-licenses:
+	 cd ${OTELCOL_BUILDER_DIR} && go install github.com/google/go-licenses@latest
```

Comment:
Consider to use a fixed version instead of latest to have deterministic builds.

Reply:
@bogdandrutu I have added latest version `v1.2.0`, Please review